### PR TITLE
Localize dragon claws spell effect

### PR DESCRIPTION
**Pull request by MSAbaddon**

### CODE REVIEW

File: static/lang/re-en.json [hunk 1]
```diff
@@ -1697,6 +1697,11 @@
                 "Deteriorated": "Deteriorated"
             }
         },
+        "SpellEffects": {
+            "DragonClaws": {
+                "Label": "Dragon Claw"
+            }
+        },
```
Comment (TikaelSol) [inline static/lang/re-en.json:1704]: This should go under SpecificRules, instead of adding a new top level key

Comment (MSAbaddon) [reply static/lang/re-en.json]: Changed

File: static/lang/re-en.json [hunk 2]
```diff
@@ -1583,6 +1583,11 @@
                 "Weaving": "Weaving",
                 "Woodworking": "Woodworking"
             },
+            "SpellEffects": {
+                "DragonClaws": {
+                    "Label": "Dragon Claw"
+                }
+            },
```
Comment (stwlam) [inline static/lang/re-en.json:1590]:
```suggestion
```

File: packs/data/spell-effects.db/spell-effect-dragon-claws.json [hunk 3]
```diff
@@ -28,8 +28,9 @@
                 },
                 "group": "brawling",
                 "key": "Strike",
-                "label": "Dragon Claw",
+                "label": "PF2E.SpecificRule.SpellEffects.DragonClaws.Labels",
```
Comment (stwlam) [inline packs/data/spell-effects.db/spell-effect-dragon-claws.json:31]:
```suggestion
                "label": "PF2E.Weapon.Base.claw",
```

File: packs/data/spell-effects.db/spell-effect-dragon-claws.json [hunk 4]
```diff
@@ -40,7 +41,7 @@
                 "damageType": "{actor|system.custom.dragonBloodline.damageType}",
                 "dieSize": "d6",
                 "key": "DamageDice",
-                "label": "Dragon Claw",
+                "label": "PF2E.SpecificRule.SpellEffects.DragonClaws.Label",
```
Comment (stwlam) [inline packs/data/spell-effects.db/spell-effect-dragon-claws.json:44]:
```suggestion
                "label": "PF2E.Weapon.Base.claw",
```

**Review by stwlam — CHANGES_REQUESTED** [review]
No need to add a new entry for this

**Pull request merged**